client/src/components/modals/affiliation-error.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <v-dialog v-model="showModal" max-width="60%">
+    <v-card class="notify-dialog">
+      <v-card-title>
+        <slot name="icon">
+          <v-icon large color="error">error</v-icon>
+        </slot>
+        <span>
+          <slot name="title">Error Adding Name Request</slot>
+        </span>
+      </v-card-title>
+      <v-card-text>
+        <slot name="text">The specified name request has already been affiliated.</slot>
```

Comment:
This is a hard-coded error message, but surely there can be other errors if "createNRAffiliation" failed?

Reply:
There is for sure. As mentioned above, building a more robust means of handling the various error codes from the various requests was some work to still be done. The scope of this ticket grew considerably, so that was something to come under another ticket.